test/Microsoft.AspNetCore.Mvc.IntegrationTests/AuthorizeFilterIntegrationTest.cs

```diff
@@ -140,6 +140,11 @@ public Task<AuthorizationPolicy> GetPolicyAsync(string policyName)
                 };
                 return Task.FromResult(new AuthorizationPolicy(requirements, new string[] { }));
             }
+
+            public Task<AuthorizationPolicy> GetRequiredPolicyAsync()
+            {
+                return Task.FromResult<AuthorizationPolicy>(null);
+            }
```

Comment:
Ok I am the confuse

Reply:
https://github.com/aspnet/Security/pull/1894#issuecomment-434985649